app/components/ak-accordion/group/index.ts

```diff
@@ -0,0 +1,81 @@
+import { action } from '@ember/object';
+import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
+
+export interface AccordionCtxProps {
+  currentOpenId: string;
+  openMultiple?: boolean;
+  handleChange: (id: string) => void;
+  accordionOpenStates: Record<string, boolean>;
+}
+
+export interface AkAccordionGroupSignature {
+  Element: HTMLElement;
+  Args: {
+    onChange?: (id: string) => void;
+
+    /**
+     * @param {Array<string>} [defaultOpen]
+     * @description Specifies default open accordion(s)
+     */
+    defaultOpen?: Array<string>;
+
+    /**
+     * @param {boolean} [openMultiple]
+     * @description Specifies whether an accordion group should open multiple accordions at the same time
+     */
+    openMultiple?: boolean;
+  };
+  Blocks: {
+    default: [AccordionCtxProps];
+  };
+}
+
+export default class AkAccordionGroupComponent extends Component<AkAccordionGroupSignature> {
+  @tracked currentOpenId = '';
+  @tracked accordionOpenStates: Record<string, boolean> = {};
+
+  constructor(owner: unknown, args: AkAccordionGroupSignature['Args']) {
+    super(owner, args);
+
+    if (!this.args.openMultiple && this.args.defaultOpen) {
+      this.currentOpenId = this.args.defaultOpen[0] || '';
+    }
+
+    if (this.args.openMultiple && this.args.defaultOpen?.length) {
+      const openStates = this.accordionOpenStates;
+      this.args.defaultOpen.forEach((group) => (openStates[group] = true));
+      this.accordionOpenStates = openStates;
```

Comment:
instead of a temporary variable this can be simplified to:
```
this.accordionOpenStates = this.args.defaultOpen.reduce((acc, curr) => ({...acc, [curr]: true}), this. accordionOpenStates);
```